src/debugulator_main.cpp

```diff
@@ -55,6 +57,11 @@ namespace
           return -1;
       }
 
+      if (vm.count(ebi::vcf::VERSION)) {
+          std::cout << "VCF-Debugulator version " << VERSION_MAJOR << "." << VERSION_MINOR << std::endl;
```

Comment:
`vcf-debugulator` to match the capitalization of the binary? (same for `vcf-validator`)